packages/f-checkout/src/components/Checkout.vue

```diff
@@ -193,10 +199,15 @@ export default {
     watch: {
         async checkoutUrl () {
             await this.loadCheckout();
+        },
+
+        authToken () {
+            this.updateAuth({ authToken: this.authToken });
         }
     },
 
     async mounted () {
+        this.updateAuth({ authToken: this.authToken });
```

Comment:
Should be an action call. You can pass that a payload in the same way and then the store will commit the mutation instead.